modules/core/src/v2/coins/trx.ts

```diff
@@ -320,13 +320,8 @@ export class Trx extends BaseCoin {
   getExtraPrebuildParams(buildParams: any, callback?: NodeCallback<any>): Bluebird<any> {
     return co<any>(function* () {
       if (buildParams.recipients[0].data && buildParams.feeLimit) {
```

Comment:
Why you are asking about `.data` if it is not used into `then` piece of code?

Reply:
Yes, i used that condition to check if the transaction is of type ContractCall

Reply:
@matilareo  confirmed that .data is used to be sure that this is a contract call.

Done.